modules/serverless-logic-verifying-workflow-deployment-preview-mode.adoc

```diff
@@ -0,0 +1,132 @@
++
+[source,yaml]
+----
+apiVersion: sonataflow.org/v1alpha08
+kind: SonataFlow
+metadata:
+  name: greeting
+  annotations:
+    sonataflow.org/description: Greeting example on k8s!
+    sonataflow.org/version: 0.0.1
+spec:
+  flow:
+    start: ChooseOnLanguage
+    functions:
+      - name: greetFunction
+        type: custom
+        operation: sysout
+    states:
+      - name: ChooseOnLanguage
+        type: switch
+        dataConditions:
+          - condition: "${ .language == \"English\" }"
+            transition: GreetInEnglish
+          - condition: "${ .language == \"Spanish\" }"
+            transition: GreetInSpanish
+        defaultCondition: GreetInEnglish
+      - name: GreetInEnglish
+        type: inject
+        data:
+          greeting: "Hello from JSON Workflow, "
+        transition: GreetPerson
+      - name: GreetInSpanish
+        type: inject
+        data:
+          greeting: "Saludos desde JSON Workflow, "
+        transition: GreetPerson
+      - name: GreetPerson
+        type: operation
+        actions:
+          - name: greetAction
+            functionRef:
+              refName: greetFunction
+              arguments:
+                message:  ".greeting+.name"
+        end: true
+----
+
+. Use `oc expose` to create a route for the workflow service as follows:
++
+[source,terminal]
+----
+$ oc expose svc/greeting -n workflows
+----
++
+This command creates a public URL to access the workflow service.
++
+
+.Example command for `greetings-workflow.yaml` file
+[source,terminal]
+----
+$ oc expose svc/greeting -n workflows
+----
+
+. Retrieve the public URL:
++
+[source,sh]
+----
+WORKFLOW_SVC=$(oc get route/<workflow-service-name> -n <namespace> --template='{{.spec.host}}')
+----
++
+This command fetches the public URL for the workflow service and stores it in the `WORKFLOW_SVC` variable.
+
++
+.Example command for `greetings-workflow.yaml` file
+[source,sh]
+----
+GREETING_SVC=$(oc get route/greeting --template='{{.spec.host}}')
+----
+
+. Make an HTTP Call to the Workflow using `curl` command to send a POST request to the service.
++
+[source,sh]
+----
+curl -X POST -H 'Content-Type: application/json' -H 'Accept: application/json' -d '{"your": "json_payload"}' http://$WORKFLOW_SVC/<endpoint>
+----
++
+.Example command for `greetings-workflow.yaml` file
+[source,sh]
+----
+curl -X POST -H 'Content-Type: application/json' -H 'Accept: application/json' -d '{"name": "John", "language": "English"}' http://$GREETING_SVC/greeting
+----
++
+This command sends a POST request to the `greeting` service with the required data.
+
+. Verify the response.
++
+If everything is working correctly, you will receive a response similar to this:
++
+.Expected response
+[source,json]
+----
+{
+  "id": "b5fbfaa3-b125-4e6c-9311-fe5a3577efdd",
+  "workflowdata": {
+    "name": "John",
+    "language": "English",
+    "greeting": "Hello from JSON Workflow, "
+  }
+}
+----
```

Comment:
Looks like example output of the previous step rather than its own step
```suggestion
+
.Example output
[source,json]
----
{
  "id": "b5fbfaa3-b125-4e6c-9311-fe5a3577efdd",
  "workflowdata": {
    "name": "John",
    "language": "English",
    "greeting": "Hello from JSON Workflow, "
  }
}
---- 
+
This output shows an example of the expected response if the workflow is running.
```